src/aabb.rs

```diff
@@ -243,6 +229,33 @@ impl<S: BaseNum> Aabb for Aabb2<S> {
     }
 }
 
+impl<S: BaseNum> Containment<Point2<S>> for Aabb2<S> {
```

Comment:
maybe, `Container`?

Reply:
Agreed, `Container` is better

Reply:
Hmm, Container to me represent something that holds other objects, I.e Vec or HashMap. I do agree that the name isn't stellar

Reply:
Cover? Envelope?

Reply:
The the hardest things in computer science :

1. Naming things 
2. Off by one
3. I forgot the name of the third thing...

Reply:
isn't it in rust guidlines to name traits with single functions as those function? i.e. `Contains`

Reply:
Can't find it in there, but that is better than the alternatives, so I'm all for it.